src/suricata.c

```diff
@@ -2928,6 +2932,10 @@ int SuricataMain(int argc, char **argv)
         /* Ignore livedev id when comparing flows. */
         g_livedev_mask = 0x0000;
     }
+    if (ConfGetBool("decoder.recursion-level.use-for-tracking", &tracking) == 1 && !tracking) {
```

Comment:
Why `decoder` ? The other fields up do not use that...

Reply:
> Why `decoder` ? The other fields up do not use that...

The decoder config namespace was suggested in the first patch revision, see https://github.com/OISF/suricata/pull/9363 . Thanks